package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lev-api",
-  "version": "0.13.0",
+  "version": "0.14.0",
```

Comment:
I think this can stay as `v0.13`. i.e. `0.13.1`.

Reply:
Done